harmony_regridding_service/dimensions.py

```diff
@@ -25,7 +25,7 @@ def horizontal_dims_for_variable(
     return GridDimensionPair(*dim_pair) if dim_pair else None
 
 
-def get_row_dims(dims: Iterable[str], var_info: VarInfoFromNetCDF4) -> str:
+def get_row_dims(dims: Iterable[str], var_info: VarInfoFromNetCDF4) -> list[str]:
```

Comment:
I updated my lsp server and it's doing typing on the fly now, so there's a few of these type fixes even though we don't enable it for the full project (I tried. Noped out)